plugins/UM3NetworkPrinting/src/Network/LocalClusterOutputDevice.py

```diff
@@ -122,9 +122,6 @@ def requestWrite(self, nodes: List[SceneNode], file_name: Optional[str] = None,
 
         self.writeStarted.emit(self)
 
-        # Make sure the printer is aware of all new materials as the new print job might contain one.
-        self.sendMaterialProfiles()
-
```

Comment:
oooh, yeah. That could have been a problem. It would certainly give our little printer more to do at the same time.

If we still need something like this, we could trigger it on the completion of the print job transmission.

Reply:
I'm mostly wondering why this would suddenly be a problem now. I'm pretty sure it has been doing this for a while now.

Reply:
It seems to be tempting fate here.

It is possible that with the new MISP firmware on the printer we've hit some kind of limit regarding CPU/memory. We'll have to do some more investigation on the printer too to cover those bases.

Reply:
To me it seams that recent changes in logging have made the problem more visible, but it's always been there (especially for large files).

Reply:
@sedwards2009 interesting thought, maybe indeed the printers are now slower in processing those requests.

Reply:
We should check that out anyway, but it sounds like the QNetworkManager gets the real blame here.

Reply:
I don't think it was the QNetworkManager. I'm running it with only the "only send materials on first connect" change and the decreased logging and I've not had the issue.

Reply:
We can revert this particular commit then to not introduce other issues